hooks/package.json

```diff
@@ -18,7 +18,7 @@
     "test:karma:watch": "karma start test/karma.conf.js --no-single-run"
   },
   "peerDependencies": {
-    "preact": "10.0.0-alpha.0"
+    "preact": "10.0.0-alpha.1"
```

Comment:
idea: relax these to caret versions?

Reply:
Good idea, there is no way to install different versions anyway.